src/Microsoft.MobileBlazorBindings.UnitTests/ParamerterParsingTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.MobileBlazorBindings.Core;
+
+namespace Microsoft.MobileBlazorBindings.UnitTests
+{
+    public class ParamerterParsingTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void ParseValidString()
```

Comment:
For this type of repetitive test, can you use something like `TestCaseData`? https://docs.nunit.org/articles/nunit/writing-tests/TestCaseData.html

Reply:
Yes I can, with this I can cover a lot more cases with a lot let code. Learning is fun. Thank you.